portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsDashboard.testcase

```diff
@@ -446,7 +446,7 @@ definition {
 		}
 
 		task ("View the cards in the Overview tab") {
-			for (var cardTitle : list "Total Individuals,Known,Anonymous,Enriched Profiles,Active Individuals,Top Interests as of Today") {
+			for (var cardTitle : list "Total Individuals,Known,Anonymous,Enriched Profiles,Active Individuals,Top Interests as of Yesterday") {
```

Comment:
Hey @joshitagaki, I think this change needs to be done on other test files, could you check and fix it for everyone?

Reply:
Done!